core/src/main/scala/kafka/server/BrokerServer.scala

```diff
@@ -188,9 +188,9 @@ class BrokerServer(
 
       info("Starting broker")
 
-      val clientMetricsReceiverPlugin = new ClientMetricsReceiverPlugin()
+      val clientMetricsTelemetryPlugin = new ClientTelemetryPlugin()
```

Comment:
nit: Name mismatch between variable and the class. I'd just use the same name, like the code did previously.